packages/website/components/alert.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import clsx from 'clsx'
+
+/**
+ * @typedef {Object} AlertProps
+ * @prop {string} [className]
+ * @prop {('top'|'bottom')} [position]
+ * @prop {('error'|'success'|'info')} [type]
+ * @prop {Number} [timer] How much seconds before onTimerEnd should be called
+ * @prop {Function} [onTimerEnd]
+ * @prop {JSX.Element} [children]
+ */
+
+const ANIMATION_DURATION = 400
+
+/**
+ *
+ * @param {AlertProps} props
+ * @returns
+ */
+export default function Alert({
+  className,
+  position = 'bottom',
+  type = 'info',
+  timer,
+  onTimerEnd,
+  children,
+}) {
+  const [hasTimerEnded, setTimerHasEnded] = useState(false)
+
+  if (timer && onTimerEnd) {
+    setTimeout(() => {
+      setTimerHasEnded(true)
+      setTimeout(onTimerEnd, ANIMATION_DURATION)
+    }, timer * 1000)
```

Comment:
Shouldn't this be done in `useEffect` or something similar?

Reply:
Not necessarily, since this doesn't affect any variables inside the component or affect the render in any way. But I'll change it to use `useEffect` to make it more reliable to future iterations :)